src/pyhf/__init__.py

```diff
@@ -155,13 +155,15 @@ def set_backend(backend, custom_optimizer=None, precision=None):
 from .workspace import Workspace
 from . import simplemodels
 from . import infer
+from . import compat
```

Comment:
@lukasheinrich I added `compat` to the public top level API, but you may want to revert this (also please revert the test change if you do as well).